bin/syclcc-clang

```diff
@@ -731,9 +747,13 @@ class hip_invocation:
     flags += [
         "-x", "hip",
         "-D__HIPSYCL_ENABLE_HIP_TARGET__",
-        "-D__HIPSYCL_CLANG__",
-        "-fplugin=" + os.path.join(self._hipsycl_lib_path, "libhipSYCL_clang.so")
+        "-D__HIPSYCL_CLANG__"
       ]
+    if sys.platform.startswith('win32'):
+      flags += ["-fplugin=" + os.path.join(self._hipsycl_bin_path, "hipSYCL_clang.dll")]
+    else:
+      flags += ["-fplugin=" + os.path.join(self._hipsycl_lib_path, "libhipSYCL_clang.so")]
```

Comment:
Maybe pull the plugin path into a property of the `config` class to avoid code duplication?